### ISSUE TITLE
Initialize net with pretrained weights of another dataset with different number of class labels

### ISSUE THREAD

**Issue by sabine1993** [dbolya/yolact#399]
Hi,
Ich want to initialize the yolact net with pretrained weights from another dataset.
The dataset has only three class lables.
The dataset I want to finetune the weights has 7 class lables.
And so the number of weights doesn't correspond and I get an error in the dimensions of the layers.

	size mismatch for prediction_layers.0.conf_layer.weight: copying a param with shape torch.Size([12, 256, 3, 3]) from checkpoint, the shape in current model is torch.Size([24, 256, 3, 3]).
	size mismatch for prediction_layers.0.conf_layer.bias: copying a param with shape torch.Size([12]) from checkpoint, the shape in current model is torch.Size([24]).
	size mismatch for semantic_seg_conv.weight: copying a param with shape torch.Size([3, 256, 1, 1]) from checkpoint, the shape in current model is torch.Size([7, 256, 1, 1]).
	size mismatch for semantic_seg_conv.bias: copying a param with shape torch.Size([3]) from checkpoint, the shape in current model is torch.Size([7]).

Is there any chance that I can somehow use the weights of the dataset with only 3 classes ?

**Comment by jasonkena** [comment]
Yup definitely. The solution is stated in https://github.com/dbolya/yolact/issues/36.

**Comment by sabine1993** [comment]
Thanks a lot. It is working !
